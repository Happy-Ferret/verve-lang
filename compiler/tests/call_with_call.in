(mul (add 2 1) 5)
