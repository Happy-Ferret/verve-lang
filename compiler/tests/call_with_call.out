push $5
push $1
push $2
push $2
call add
push $2
call mul
